### DISCUSSION TITLE
Overriding plugin config with theme.color is not working

### DISCUSSION THREAD

**Discussion by whanso** [tailwindlabs/tailwindcss#7735] [Help]
Hello,

I am working on my tailwind.config.js file that incorporates a plugin.

```
const defaultTheme = require('tailwindcss/defaultTheme');

/** @type {import('tailwindcss/tailwind-config').TailwindConfig} */
module.exports = {
  content: ['./src/**/*.{js,ts,jsx,tsx}', './node_modules/flowbite/**/*.js'],
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      white: '#fff',
      gray: {
        100: '#B1B3B399',
        200: '#B1B3B3CC',
        300: '#B1B3B3',
      },
    },
  },
  plugins: [require('flowbite/plugin')],
};
```

Using the `colors` object in  `tailwind.config.js`, I was expecting usages of `gray-300` in my app to be the value I provided, but instead it is picked up as the value in the plugin.

Here is the unpkg of the plugin: [https://unpkg.com/flowbite@1.3.4/plugin.js](https://unpkg.com/flowbite@1.3.4/plugin.js)

**Expected result**: The configuration for `tailwind.config.js` should take priority of the a plugin configuration.  The flowbite documentation states that the colors can be overridden and replaced.
**Actual result**: The plugin configuration takes priority over the `tailwind.config.js` theme.

** Wrong Computed Value (equates to #d1d5db, from Flowbite plugin): **
<img width="480" alt="Screen Shot 2022-03-03 at 10 40 34 AM" src="https://user-images.githubusercontent.com/26184716/156598639-b8ffc999-5506-485e-8a24-6d321b2d7086.png">

OS: MacOS

**Comment by jeff-treuting** [comment]
I'm running into this problem as well.  Is there a workaround?

**Reply by wongjn** [reply]
You could define your colors in `theme.extend.colors`, see [this Tailwind Play](https://play.tailwindcss.com/HT08lA6Nn6?file=config) (open the *Generated CSS* panel).

**Reply by jeff-treuting** [reply]
that works, thank you.  I thought extend was only for new colors, so I had the colors in the theme, which worked except for when the flowbite plugin was set.

thanks for the quick response.
